### DISCUSSION TITLE
When Wiki.js will be available in version 3.0?

### DISCUSSION THREAD

**Discussion by YunusEmreAlps** [requarks/wiki#5359] [Help / Questions] [answered]
Hello,

I hope you're well and everything is good for you.

When WikiJS will be available in version 3.0? 

I’m curious about the wiki.js project and I'm working on this project right now. I add this project a couple of features. 
How can I access version 3.0? 

Thank you very much.
Best regards,

**Comment by 573077632** [comment]
Not yet ready to release beta.

**Reply by 573077632** [reply]
In the branch library vega, there is V3.0 content.

**Reply by YunusEmreAlps** [reply]
Thank You

**Reply by YunusEmreAlps** [reply]
When I was running the vega branch, I'm facing a database connection error.
How can I run this vega branch?

**Reply by 573077632** [reply]
I don't know how to run Vega branch, and I encountered a database error when I tried it myself

**Reply by YunusEmreAlps** [reply]
@573077632 Did you know this repo? 
V3 repository: https://github.com/Requarks/wiki-vnext.

**Reply by 573077632** [reply]
This library has been abandoned.

**Reply by YunusEmreAlps** [reply]
I was able to run the UX part of this project. When I ran this repository I saw the new design. I want to work with 3.0 but I guess this not gonna be happening.

**Reply by 573077632** [reply]
耐心等待几个月吧，最近快要出beta了

**Comment by NGPixel** [comment, marked as answer]
v3 is not ready yet and help will not be provided for it until the beta is ready.

See https://blog.js.wiki/news/2022/the-road-to-the-wiki-js-3-beta
